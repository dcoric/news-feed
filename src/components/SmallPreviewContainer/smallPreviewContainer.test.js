import React from 'react';
import { shallow } from 'enzyme';
import { SmallPreviewContainer } from '../index';

const smallPreviewContainer = shallow(<SmallPreviewContainer />);

describe('SmallPreviewContainer', () => {
  describe('Rendering snapshot', () => {
    it('renders correctly', () => {
      expect(smallPreviewContainer).toMatchSnapshot();
    });
  });
});
