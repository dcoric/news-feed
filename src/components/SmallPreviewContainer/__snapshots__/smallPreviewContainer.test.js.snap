// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallPreviewContainer Rendering snapshot renders correctly 1`] = `ShallowWrapper {}`;
